common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -43,7 +43,7 @@ def is_browser_on_page(self):
         """
         Check if browser is showing correct page.
         """
-        return self.q(css='body.view-profile').present
+        return self.q(css='body.view-profile .account-settings-container').present
```

Comment:
I'm not 100% clear on the context of this line change, but I'm assuming the tests pass with it, so carry on.